### DISCUSSION TITLE
Changing theme

### DISCUSSION THREAD

**Discussion by rasii** [tinkerun/tinkerun#16] [General]
I've got this installed and running on windows and it's awesome.  My only issue at the moment is the theme.  I only see the one to select and error messages are almost impossible to read.  Of course, I should never have errors, but you know... :)

Are there other themes available and I'm not seeing them?  Is there anything I can do to make errors more readable?

Thanks for an awesome alternative to tinkerwell.

After the issues reading error messages I went ahead and bought tinkerwell - it's actually harder to set up and use than tinkerun - and I have an issue that I can't even type in the code window - I'm trying to get support from them now :(

**Comment by billyct** [comment]
thx for you like it.

I haven't really made the theme done yet, now there are only two themes `default` and `classic`, it all comes from which the [UI lib](https://evergreen.segment.com/) I'm using, and the difference is not big, just made a simple interface, and then I will improve it when I have time.

and if you have some errors, you can [create an issue](https://github.com/tinkerun/tinkerun/issues/new) for it, the more detailed the better. 

I think there must be a lot of bugs, especially windows 😂.

**Reply by rasii** [reply]
Thanks for the quick reply.  The errors I mentioned are errors in my php code, when the code runs with errors, the errors displayed by your UI are black text with red background.  On my system it's almost impossible to read.  Making that more readable might be easy and would make a HUGE difference for me.
![Capture](https://user-images.githubusercontent.com/6601063/119540823-7ce13000-bd53-11eb-8240-86db15b0a5c9.PNG)

**Reply by billyct** [reply]
You’re right, it looks really bad on windows, it looks okay on mac, I think I’m going to solve these problems in the next versions.

I think if I can have a way to make [these vscode themes](https://vscodethemes.com/) compatible, it will be great.
